omrsigcompat/makefile

```diff
@@ -25,6 +25,18 @@ OBJECTS := omrsig
 OBJECTS := $(addsuffix $(OBJEXT),$(OBJECTS))
 EXPORT_FUNCTIONS_FILE := omrsig.exportlist
 
+ifeq (win,$(OMR_HOST_OS))
```

Comment:
So that means, omr.res has to be generated before the compilation stage. I don't think any change in configure stuff will help us out coz configure needs to tell the order:  1) first write to omr.rc from omr.rc.in and 2) then generate omr.res from omr.rc if exists.

Otherwise, you have to specify the action in all makefiles where required to link to .dll file.